python_modules/dagster-graphql/dagster_graphql/schema/sensors.py

```diff
@@ -171,6 +176,24 @@ def mutate(self, graphene_info, job_origin_id):
         return stop_sensor(graphene_info, job_origin_id)
 
 
+class GrapheneSetSensorCursorMutation(graphene.Mutation):
+    Output = graphene.NonNull(GrapheneSensorOrError)
+
+    class Arguments:
+        sensor_selector = graphene.NonNull(GrapheneSensorSelector)
+        cursor = graphene.String()
+
+    class Meta:
+        name = "SetSensorCursorMutation"
+
+    @capture_error
+    @check_permission(Permissions.SET_SENSOR_CURSOR)
```

Comment:
this seems very granular, should we have a generic edit_sensors permission? hard to imagine letting somebody turn the sensor on and off but not mess with the cursor